### PR TITLE
Fix for allowing multiple marks when using the FreehandLine tool conn…

### PR DESCRIPTION
**Pull request by kieftrav**
## Describe your changes
The FreehandLine tool needs to be able to display multiple marks from Caesar. The changes are minimal and are a refactor to allow multiple marks to be processed. 

## How to Review
- Run the branch locally and ensure that the [Staging Freehand Line Project](https://localhost:8080/?project=kieftrav/freehand-line-test&env=staging) loads two marks.

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [ ] Tests are passing locally and on Github
- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
- [ ] All pages of a FEM project load: Home Page, Classify Page, and About Pages
- [ ] Can submit a classification
- [ ] Can sign-in and sign-out
- [ ] The component is accessible
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

## Bug Fix
- [ ] The PR creator has listed user actions to use when testing if bug is fixed
- [ ] The bug is fixed
- [ ] Unit tests are added or updated

### CODE REVIEW

File: packages/lib-classifier/src/store/subjects/Subject/FreehandLineReductions/FreehandLineReductions.js [hunk 1]
```diff
@@ -15,29 +15,31 @@ const FreehandLineReductions = types
...
-
-          if (data.frame === frame
-            && data.stepKey === stepKey
-            && data.taskKey === task.taskKey
-            && data.taskType === task.type
-            && data.toolType === tool.type
-          ) {
-            caesarMark = data
-          }
+          let { data } = reduction.data   // Caesar expects data attribute to be an object
```
Comment (eatyourgreens) [inline packages/lib-classifier/src/store/subjects/Subject/FreehandLineReductions/FreehandLineReductions.js:26]: Should this be: 
```suggestion
          let { data } = reduction   // Caesar expects data attribute to be an object
```

File: packages/lib-classifier/src/store/subjects/Subject/FreehandLineReductions/FreehandLineReductions.spec.js [hunk 2]
```diff
@@ -35,15 +35,18 @@ describe('Models > FreehandLineReductions', () => {
...
 
   it('should have array of x and y values', () => {
-    let caesarMark = reductionsModel.findCurrentTaskMark(reductionsTaskStub)
-    expect(caesarMark.pathX).to.be.a('array')
-    expect(caesarMark.pathY).to.be.a('array')
-    expect(caesarMark.pathX).not.to.be.empty()
-    expect(caesarMark.pathY).not.to.be.empty()
+    let caesarMarks = reductionsModel.findCurrentTaskMarks(reductionsTaskStub)
+    expect(caesarMarks).to.be.a('array')
+    expect(caesarMarks[0].pathX).to.be.a('array')
```
Comment (eatyourgreens) [inline packages/lib-classifier/src/store/subjects/Subject/FreehandLineReductions/FreehandLineReductions.spec.js:47]: Whitespace?

**Review by eatyourgreens — APPROVED** [review]
Changes look good on a read through. I'm happy to merge this and iterate.

File: packages/lib-classifier/src/store/subjects/Subject/FreehandLineReductions/FreehandLineReductions.js (hunk 1 above)
Comment (kieftrav) [reply packages/lib-classifier/src/store/subjects/Subject/FreehandLineReductions/FreehandLineReductions.js:26]: This is a silly thing where it's reduction.data.data = [] (it's a comment that Caesar cannot have reduction.data = []). Might be better to rename to reduction.data.marks or something similar.

**Pull request merged**